app/src/main/res/values/strings.xml

```diff
@@ -20,6 +20,8 @@
     <string name="action_previous">Previous</string>
     <string name="action_next">Next</string>
 
+    <string name="pref_title_enqueue_song_default_choice">Enqueue song default action</string>
```

Comment:
I think this should rather be called "Default song action". I think [enqueue](https://en.wiktionary.org/wiki/enqueue) is only one of these options (the last one shown in the screenshot).

Reply:
How about "Default action on choosing/tapping a song while browsing"